multichain-testing/test/README.md

```diff
@@ -0,0 +1,20 @@
+## Start Chains
+
+Runs agoric, cosmoshub, and osmosis with hermes relayers.
+
+```sh
+# start starship with default configuration
+make start
+```
+
+## Run Tests
+
+```sh
+yarn test:main
```

Comment:
this suggests `main` is what you're supposed to pick. This README should explain that the tests needs selectors and how to pick